adapters/appnexus/appnexus.go

```diff
@@ -216,6 +200,27 @@ func (a *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest
 	return bidderResponse, errs
 }
 
+func (a *adapter) addRequestLevelExt(ext json.RawMessage, isAMP int, isVIDEO int) (appnexusReqExt, error) {
```

Comment:
Thoughts about changing the name of this function to `getRequestExt`? The prefix `add` sounds a little misleading because we aren't really adding the request ext to the request in this function. We're just generating the appnexus request ext and returning that. 

I considered naming it `getAppnexusReqExt` but we're already considering renaming `appnexusReqExt` as the `appnexus` prefix is kinda redundant so similar logic applies here too. However, I can then also understand the confusion it can lead to between openrtb req ext vs appnexus req ext so I will leave choosing between the two upto you.

Reply:
Changed it to `getBidderRequestExt` in [410f953](https://github.com/prebid/prebid-server/pull/2822/commits/410f95318e4827546656df19bb51997dada9c1e0)